WinEventParser/main.py

```diff
@@ -69,39 +69,50 @@ def convert_time_in_utc(time_local_timezone):
     return time_utc
 
 
+def get_num_of_timestamps(input_mode_timestamp):
+    """Function used to get the number of timestamps the user wants to input"""
+    if input_mode_timestamp == "two":
+        return 2
+    return 1
+
+
+def get_input_mode(iteration_of_for_loop):
+    """Function used to get the input mode for a timestamp"""
+    mode_to_pass = ""
+    mode_to_check = ["1", "2", "3", "4", "5", "6"]
+    while mode_to_pass not in mode_to_check:
+        mode_to_pass = input(f"[{iteration_of_for_loop}] Please when you input your timestamp do you want to input 1) a full timestamp, 2) a timestamp with date, hour and minute, 3) a timestamp with date and hour, 4) a timestamp with date only, 5) a timestamp with year and month, 6) a timestamp with year (1/2/3/4/5/6): ")
+    return int(mode_to_pass)
+
+
 def get_input_timestamp(input_mode_timestamp):
```

Comment:
Looking better. Still needs some refactoring.

Reply:
Yeah totally ! Just I was more focused on the first one you assigned but for sure I'll modify it ASAP